dll/win32/shell32/lang/tr-TR.rc

```diff
@@ -4,7 +4,7 @@
  * PURPOSE:     Turkish resource file
  * TRANSLATORS: Copyright 2006 Fatih Aşıcı <fasici@linux-sevenler.org>
  *              Copyright 2012 Arda Tanrıkulu (ardatan) <ardatanrikulu@gmail.com>
- *              Copyright 2015-2016, 2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
+ *              Copyright 2015-2020 Erdem Ersoy (eersoy93)
```

Comment:
Please do not remove email even if it's outdated information. (I'm fine with correcting the year, but please also make it a separate commit in master)

Reply:
I do see no reason why this single-line-comment-change should be a separate commit. You do mix such header changes into everybody elses PRs all the time. You are absolutely applying double-standards to how you work and to how you do expect anybody else to work.

Reply:
It's not about a separate commit, it is about you removing the email address.